Source/Blazorise/wwwroot/dropdown.js

```diff
@@ -44,19 +47,23 @@ export function initialize(element, elementId, targetElementId, altTargetElement
         : element.querySelector(createSelector(options.dropdownMenuClassNames));
 
     const instance = createPopper(targetElement, menuElement, {
-        placement: getPopperDirection(options.direction),
-        strategy: "fixed",
+        placement: getPopperDirection(options.direction, options.rightAligned),
+        strategy: "absolute",
         modifiers: [
             {
                 name: "preventOverflow",
+                enabled: true,
                 options: {
                     padding: 0,
                 }
-            }]
+            }
+        ]
     });
 
     instance.update();
 
+    createAttributesObserver(menuElement.id, (mutationsList) => observeClassChanged(mutationsList, options.dropdownShowClassName, () => instance.update()));
```

Comment:
What if `options.dropdownShowClassName` is null?

Reply:
then it certainly looks for a null class which will never find I guess. I'll add a null check with an early return in the observer